docs/samples/Microsoft.ML.AutoML.Samples/MulticlassClassificationExperiment.cs

```diff
@@ -46,7 +56,7 @@ public static void Run()
                 mlContext.Model.Save(bestRun.Model, trainDataView.Schema, fs);
 
             // STEP 6: Create prediction engine from the best trained model
-            var predictionEngine = mlContext.Model.CreatePredictionEngine<PixelData, PixelPrediction>(bestRun.Model);
+             var predictionEngine = mlContext.Model.CreatePredictionEngine<PixelData, PixelPrediction>(bestRun.Model);
```

Comment:
I'm not sure if we want changes in sample get checked in

Reply:
please look at iteration #4

---
In reply to: [339325057](https://github.com/dotnet/machinelearning/pull/4395#discussion_r339325057) [](ancestors = 339325057)